src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -23,8 +24,11 @@ export class OddOneOutComponent implements OnInit {
     private _subscription;
     private _data;
     private _status: object = {stage: 2, activity: 3};
+    private readonly pointsPerCorrectAnswer = 10;
```

Comment:
follow the CAPITALIZED_WITH_UNDERSCORES naming convention for all static and readonly constants/variable .